pkg/controller/user/export.go

```diff
@@ -51,17 +59,42 @@ func (c *Controller) HandleExport() http.Handler {
 			return
 		}
 
-		w.Header().Add("Content-Disposition", "")
-		w.Header().Add("Content-Type", "text/CSV")
+		filename := fmt.Sprintf("%s-users.csv", time.Now().Format(project.RFC3339Squish))
+		c.h.RenderCSV(w, 200, filename, membershipsCSV(memberships))
+	})
+}
+
+// membershipsCSV is a CSV writer.
+type membershipsCSV []*database.Membership
 
-		csvWriter := csv.NewWriter(w)
-		for _, membership := range memberships {
-			user := membership.User
-			if err := csvWriter.Write([]string{user.Email, user.Name}); err != nil {
-				logger.Errorw("error writing record to csv:", "error", err)
-				break
-			}
+// MarshalCSV returns bytes in CSV format.
+func (s membershipsCSV) MarshalCSV() ([]byte, error) {
+	// Do nothing if there's no records
+	if len(s) == 0 {
+		return nil, nil
+	}
+
+	var b bytes.Buffer
+	w := csv.NewWriter(&b)
+
+	if err := w.Write([]string{"name", "email", "joined"}); err != nil {
+		return nil, fmt.Errorf("failed to write CSV header: %w", err)
+	}
+
+	for i, m := range s {
+		if err := w.Write([]string{
+			m.User.Name,
+			m.User.Email,
+			m.CreatedAt.Format(project.RFC3339Date),
```

Comment:
Wondering: why add CreatedAt - presumably when we import these we want a fresh CreatedAt?

Reply:
I think we would discard that value on import (we don't consume it). This is for if someone wants to know _when_ someone was added to a realm.